src/py/pyodide/_package_loader.py

```diff
@@ -26,12 +26,18 @@
 if sys.base_prefix == sys.prefix:
     # not in a virtualenv
     STD_LIB = Path(sysconfig.get_path("stdlib"))
-    TARGETS = {"site": SITE_PACKAGES, "lib": STD_LIB, "dynlib": Path("/usr/lib")}
+    DSO_DIR = Path("/usr/lib")
 else:
     # in a virtualenv
-    # Better not put stuff into /usr/lib or /lib/python3.10! For now let's stick
-    # everyone into SITE_PACKAGES in this case
-    TARGETS = {"site": SITE_PACKAGES, "lib": SITE_PACKAGES, "dynlib": SITE_PACKAGES}
+    # Better not put stuff into /usr/lib or /lib/python3.10! Stick the stdlib
+    # into SITE_PACKAGES and dsos up two directories.
+    #
+    # e.g., SITE_PACKAGES = .venv/lib/python3.10/site_packages
+    # and   DSO_DIR       = .venv/lib/
+    STD_LIB = SITE_PACKAGES
+    DSO_DIR = SITE_PACKAGES.parents[1]
```

Comment:
Could you check if `import shapely` works in cmdline runner? I think is always try to load `geos` from `/usr/lib`.

Reply:
I suspect it doesn't work.

Reply:
#3167 would (hopefully) help fix the `shapely` import issue.